src/gui/Src/Gui/CPUStack.cpp

```diff
@@ -406,7 +424,80 @@ QString CPUStack::paintContent(QPainter* painter, dsint rowBase, int rowOffset,
         painter->drawText(QRect(x + 4, y , w - 4 , h), Qt::AlignVCenter | Qt::AlignLeft, makeAddrText(wVa));
         return QString();
     }
-    return HexDump::paintContent(painter, rowBase, rowOffset, col, x, y, w, h);;
+    else if(col == 1) // paint stack data
+    {
+        // get call stack
+        DBGCALLSTACK callstack;
+        memset(&callstack, 0, sizeof(DBGCALLSTACK));
+        // TODO: need to cache the call stack data.
+        DbgFunctions()->GetCallStack(&callstack);

```

Comment:
In fact I have not observed a significant performance loss. After all there is only a few lines in the CPUStack that need to repaint.